### ISSUE TITLE
additionalDependencies not loaded correctly

### ISSUE THREAD

**Issue by mashehu** [pre-commit/mirrors-prettier#46]
Since 3.0.0, I get the error:
`Cannot find package 'prettier-plugin-astro' imported from $PATH_TO_MY_REPO/noop.js`

My `pre-commit-config.yaml`:

```yaml
repos:
  - repo: https://github.com/pre-commit/mirrors-prettier
    rev: 'v3.0.0'
    hooks:
      - id: prettier
        additional_dependencies:
          - prettier@3.0.0
          - prettier-plugin-astro@^0.11.0
        args: [ --plugin=prettier-plugin-astro]
        files: \.(astro)$
```

Works fine if I use `v.2.7.1`.

**Comment by asottile** [comment]
there's nothing special about this repo it just calls prettier

**Comment by brownan** [comment]
I just ran into this too. I suspect it's something with the way pre-commit sets up the nodejs environment is incompatible with the way that prettier tries to import its plugins. It's not looking in the right place or something.

As a workaround, I found I can put the relative path to the plugin instead of just the plugin name. I specified in my package.json:

```
prettier": {
        "tabWidth": 4,
        "printWidth": 90,
        "plugins": [
            "./node_modules/prettier-plugin-organize-imports/index.js"
        ]
    }
```

Instead of the plugin name just being `prettier-plugin-organize-imports` and that seems to work. Should be able to make the same change in your pre-commit args list.
